### ISSUE TITLE
Potential legal issues with that package name

### ISSUE THREAD

**Issue by hybridherbst** [RageAgainstThePixel/com.google.fonts.roboto#3]
# Bug Report

## Overview
Saw this randomly in the OpenUPM RSS feed.
While I think it's nice to have font packages, I can only advise to stick to non-misleading package names, that is, when you publish this as package you should name it "com.somenamethatisnotgoogle..." and properly specify who the author of the package is.

Otherwise it's legally pretty questionable, both for Unity (giving them more fodder for why scoped registries are bad), and Google who won't like the unauthorized use of their name either.

Cc'ing @favoyang

**Comment by StephenHodgson** [comment]
@hybridherbst This is [directly using the material design icon github repo](https://github.com/RageAgainstThePixel/com.google.fonts.roboto/tree/development/submodules) and the assets directly in it. Only additional stuff is the TMPro SDF assets.

Otherwise it's pretty much from google, just repackaged as upm.

If you guys would like me to change this, that's fine, but I have every intention of letting google take ownership of this repo by xfering the project to their org once it was complete. (same with a few other packages).

AFAIK, as long as they're attributed and keep ownership I don't see any issue here. All the fonts are open source.

**Comment by StephenHodgson** [comment]
@hybridherbst I'm going to rename to `com.fonts.*` for all the open source fonts

**Comment by hybridherbst** [comment]
Just wanted to raise it as concern - if you feel that's legally correct and/or have talked to Google that's totally fine.

I think Unity won't like something that's not endorsed by Google showing up as "author: Google" in PackMan. 

Just for my understanding - if you say "every intention to let Google take ownership" does that mean you're working with them on this? I like the idea of fonts as asset packages.

Future idea: it should be possible to generate font packages in a backend so that all Google fonts are available as package directly (similar to our package installer)

**Comment by StephenHodgson** [comment]
Well there's a few packages in there not authored by them showing up. Just sayin.

>  it should be possible to generate font packages in a backend so that all Google fonts are available as package directly (similar to our package installer)

Yeah it's possible but text mesh pro is being a bit difficult. I tried to do it programmatically for the com.fonts.material-icons package but I gave up on it last night.

> Just for my understanding - if you say "every intention to let Google take ownership" does that mean you're working with them on this?

I had reached out before starting the project but was still waiting for a reply. I've always been in the camp of do the thing, then ask for permission 😅 I'm not interested in making money off this stuff, just making it more widely available in a form I can utilize in other projects.

To be honest I think unity is just going overboard with the whole thing with the package manager. If this is a concern then tools like https://github.com/xoofx/UnityNuGet would also technically violate this same TOS.
